store/sqlstore/post_store.go

```diff
@@ -560,8 +509,6 @@ func (s *SqlPostStore) GetPostsSince(options model.GetPostsSinceOptions, allowFr
 		}
```

Comment:
all the references to latestUpdate before this line in this function should be removed, are only cache related so you don't need it here now.